.golangci.yml

```diff
@@ -5,7 +5,7 @@ linters:
     - goerr113
     - errcheck
     - goimports
-    - paralleltest
+    # - paralleltest # missing the call to method parallel, but testify does not seem to work well with parallel test: https://github.com/stretchr/testify/issues/187
```

Comment:
@MichaelSnowden i am disabling the paralleltest linter since this will not work with testify, see above comments